jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PushImageStep.java

```diff
@@ -97,14 +97,10 @@ public BuildResult call() throws ExecutionException, InterruptedException {
         .addSteps(NonBlockingSteps.get(pushApplicationLayersStep))
         .addStep(NonBlockingSteps.get(pushContainerConfigurationStep))
         .addStep(NonBlockingSteps.get(buildImageStep))
-        .whenAllSucceed(this::afterPushSteps)
-        .get()
-        .get()
-        .get();
+        .whenAllSucceedBlocking(this::afterPushSteps);
```

Comment:
I'm a little lost as to how this triple get() is satisfied by the single get() in whenAllSucceedBlocking